client/my-sites/theme/live-preview-button/index.tsx

```diff
@@ -16,12 +17,18 @@ interface Props {
  */
 export const LivePreviewButton: FC< Props > = ( { themeId, siteId } ) => {
 	const translate = useTranslate();
+	const dispatch = useDispatch();
+
 	const isLivePreviewSupported = useSelector( ( state ) =>
 		getIsLivePreviewSupported( state, themeId, siteId )
 	);
-	const livePreviewUrl = useSelector( ( state ) => getLivePreviewUrl( state, themeId, siteId ) );
 	if ( ! isLivePreviewSupported ) {
 		return null;
 	}
-	return <Button href={ livePreviewUrl }>{ translate( 'Live Preview' ) }</Button>;
+
+	return (
+		<Button onClick={ () => dispatch( livePreview( themeId, siteId ) ) }>
+			{ translate( 'Live Preview' ) }
```

Comment:
Nit: I think this should be `Live preview` with lowercase `p` to be consistent with the rest of the site 😄